python/ql/test/experimental/meta/ConceptsTest.qll

```diff
@@ -0,0 +1,33 @@
+import python
+import experimental.dataflow.DataFlow
+import experimental.semmle.python.Concepts
+import TestUtilities.InlineExpectationsTest
+
+string value_from_expr(Expr e) {
+  // TODO: This one is starting to look like `repr` predicate from TestTaintLib
+  result =
+    e.(StrConst).getPrefix() + e.(StrConst).getText() +
+      e.(StrConst).getPrefix().regexpReplaceAll("[a-zA-Z]+", "")
+  or
+  result = e.(Name).getId()
+  or
+  not e instanceof StrConst and
+  not e instanceof Name and
+  result = e.toString()
+}
+
+class SystemCommandExecutionTest extends InlineExpectationsTest {
+  SystemCommandExecutionTest() { this = "SystemCommandExecutionTest" }
+
+  override string getARelevantTag() { result = "SystemCommandExecution_getCommand" }
```

Comment:
I feel like this tag is _way_ too long, and that this hampers readability in the test file. Why not just `getCommand`?